vagrant/Vagrantfile

```diff
@@ -500,6 +557,38 @@ Vagrant.configure('2') do |config|
                           mount -t lustre /dev/mapper/mpathj /mnt/ost2-4
                          SHELL
 
+        oss.vm.provision 'create-ldiskfs-lvm-fs',
+                         type: 'shell',
+                         run: 'never',
+                         path: 'scripts/create_ldiskfs_fs_osts.sh',
+                         args: ['a', 'e', 0, 'fs']
+
+        oss.vm.provision 'mount-ldiskfs-lvm-fs',
+                         type: 'shell',
+                         run: 'never',
+                         inline: <<-SHELL
+                          mkdir -p /mnt/ost{0,1,2,3,4}
+                          mount -t lustre /dev/mapper/mpatha /mnt/ost0
+                          mount -t lustre /dev/mapper/mpathb /mnt/ost1
+                          mount -t lustre /dev/mapper/mpathc /mnt/ost2
+                          mount -t lustre /dev/mapper/mpathd /mnt/ost3
+                          mount -t lustre /dev/mapper/mpathe /mnt/ost4
+                         SHELL
+
+        oss.vm.provision 'ha-ldiskfs-lvm-fs-step2',
+                         type: 'shell',
+                         run: 'never',
+                         inline: <<-SHELL
+                           pcs cluster auth oss1.local oss2.local -u hacluster -p lustre
+                           pcs cluster setup --start --name mds-cluster oss1.local oss2.local --enable --token 17000
+                           pcs stonith create vboxfence fence_vbox ipaddr=10.0.2.2 login=root passwd=lustre
```

Comment:
Maybe worth noting that this won't function on a local install (unless the user has lustre as their host pw).

Reply:
I wasn't sure what to do about the vbox fence